Formula/lf.rb

```diff
@@ -20,6 +21,9 @@ def install
     cd "src/github.com/gokcehan/lf" do
       system "./gen/build.sh", "-o", bin/"lf"
       prefix.install_metafiles
+      man1.install "lf.1"
+      zsh_completion.install "etc/lf.zsh" => "_lf"
+      fish_completion.install "etc/lf.fish"
```

Comment:
Is there no bash completion?

Reply:
No, the authors have not written a bash completion file yet.

Reply:
It's nice to see this the other way around for once - it's usually bash that comes first!